frontend/javascripts/oxalis/model/sagas/volumetracing_saga.tsx

```diff
@@ -463,7 +462,9 @@ export function* floodFill(): Saga<void> {
       yield* put(
         addUserBoundingBoxAction({
           boundingBox: coveredBoundingBox,
-          name: getBBoxNameForPartialFloodfill(oldSegmentIdAtSeed, activeCellId, seedPosition),
+          name: `Limits of flood-fill (source_id=${oldSegmentIdAtSeed}, target_id=${activeCellId}, seed=${seedPosition.join(
```

Comment:
I am correctly assuming that this code is for the local, frontend only flood-fill operation?

Reply:
Exactly :+1: